cachet_url_monitor/configuration.py

```diff
@@ -184,30 +184,30 @@ def push_metrics(self):
                                     (metrics_request.status_code,))
 
     def push_incident(self):
-        if hasattr(self, 'incident_id') and self.status == 1:
-            # If the incident already exists, it means it's unhealthy. We only update it when it becomes healthy again.
-            params = {'status': 4, 'visible': 1, 'component_id': self.data['cachet']['component_id'],
-                      'component_status': self.status, 'notify': True}
+        """If the component status has changed, we create a new incident (if this is the first time it becomes unstable)
+        or updates the existing incident once it becomes healthy again.
+        """
+        if hasattr(self, 'incident_id') and self.status == cachet_url_monitor.status.COMPONENT_STATUS_OPERATIONAL:
+            # If the incident already exists, it means it was unhealthy but now it's healthy again.
+            params = {'status': 4, 'visible': 1, 'component_id': self.component_id, 'component_status': self.status,
```

Comment:
We actually verify if the member exists in the line 187, with `if hasattr(self, 'incident_id')`.